src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -104,26 +104,13 @@ for further details.
   & h5,
   & h6 {
     /*
-    These ::before elements account for the height of the top sticky bars
-    when following #anchor-links to headings. Without the offset caused
-    by these elements, we'd jump to headings that would be hidden underneath
-    the top sticky bars.
+    Because not all jump link headings have an anchor child, such as the ones on
+    the product landing pages
     */
-    & a:global(.__target-h)::before,
-    &::before {
-      --permalink-scroll-offset: 24px;
-      --full-header-height: calc(
-        var(--navigation-header-height) + var(--alert-bar-height)
-      );
-      --height: calc(
-        var(--full-header-height) + var(--permalink-scroll-offset)
-      );
-
-      display: block;
-      content: '\200B';
-      height: var(--height);
-      margin-top: calc(-1 * var(--height));
-      visibility: hidden;
+    scroll-margin-top: 11rem; /* 176px */
```

Comment:
Do we still want to use those CSS vars here? That way the offset should change if we adjust the `--navigation-header-height` or the `--alert-bar-height`.

Reply:
Hmm good call. Let me work on that. 😊